Packs/Claroty/Integrations/Claroty/Claroty.yml

```diff
@@ -0,0 +1,312 @@
+category: Authentication
```

Comment:
please change category

Reply:
To what?

Reply:
These are the categories we support today. My take would be "Network Security", because we don't have something OT/IOT specific yet. But double check with Daniel.

INTEGRATION_CATEGORIES = ['Analytics & SIEM', 'Utilities', 'Messaging', 'Endpoint', 'Network Security',
                          'Vulnerability Management', 'Case Management', 'Forensics & Malware Analysis',
                          'IT Services', 'Data Enrichment & Threat Intelligence', 'Authentication', 'Database',
                          'Deception', 'Email Gateway']